addons/website_livechat/views/website_livechat.xml

```diff
@@ -1,11 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="assets_backend" inherit_id="website.assets_backend" name="website_livechat assets backend">
-            <xpath expr="." position="inside">
-            </xpath>
-        </template>
```

Comment:
What is the reason to remove this? I think this will require a migration to reach master, shouldn't remove this kind of data in stable.

Reply:
It was bad as it's defined in another .xml of the module leading to the fact that only one of the two definitions was used (Didn't check if it was the last loaded or the other). This was the one that was used and as it was empty, no file was actually loaded in the module ..

Reply:
Which xml/other definition are you referring to? Isn't this just an override that does nothing?

Reply:
Ok I get it, you meant the `id="assets_backend"` fine to keep this diff then.

Reply:
Yeah `website_livechat/views/assets.xml` has already this assets bundle definition